index.js

```diff
@@ -195,21 +187,16 @@ let Fixtures = {
      * @return {Promise} The ChakramResponse from the creation of the records and/or links
      */
     create(models, options = {}) {
-        if (_.isUndefined(this._headers['OAuth-Token'])) {
-            if (++this._sessionAttempt > this._maxSessionAttempts) {
-                throw new Error('Max number of login attempts exceeded!');
-            }
-
-            return this._adminLogin().then(() => this.create(models, options));
+        if (this._needsLogin()) {
+            return this._adminLogin().then(() => this.create(models, options)).catch((response) => {
+                throw response;
```

Comment:
Explicit `catch` is intended to prevent unsuccessful requests from getting to `this.create`.